### PR TITLE
JuiceFS DataMigrate supports PVC

### PR DESCRIPTION
**Pull request by TrafalgarZZZ**
### Ⅰ. Describe what this PR does
JuiceFS DataMigrate supports `pvc://` volume scheme

### Ⅱ. Does this pull request fix one issue?

fixes #XXXX

### Ⅴ. Special notes for reviews

### CODE REVIEW

**Comment by cheyang** [comment]
@TrafalgarZZZ could you please fix `ineffectual assignment to pvcName` issue?

File: pkg/ddc/juicefs/data_migrate.go [hunk 1]
```diff
@@ -204,47 +204,83 @@ func (j *JuiceFSEngine) genDataUrl(data datav1alpha1.DataToMigrate, info *cdatam
...
+					VolumeSource: corev1.VolumeSource{
+						PersistentVolumeClaim: &corev1.PersistentVolumeClaimVolumeSource{
+							ClaimName: pvcName,
+						},
+					},
+				})
+				info.NativeVolumeMounts = append(info.NativeVolumeMounts, corev1.VolumeMount{
+					Name:      volName,
+					SubPath:   subPath,
+					MountPath: "/mnt/native/",
```
Comment (cheyang) [inline pkg/ddc/juicefs/data_migrate.go:236]: how about using `/mnt/fluid-native`? And put it into a constant?

Comment (TrafalgarZZZ) [reply pkg/ddc/juicefs/data_migrate.go]: Done

**Review by cheyang — APPROVED** [review]
/lgtm
/approve

**Pull request merged**